fiftyone/core/odm/database.py

```diff
@@ -848,51 +845,79 @@ def patch_saved_views(dataset_name, dry_run=False):
         return
 
     dataset_id = dataset_dict["_id"]
-    saved_views = dataset_dict.get("saved_views", [])
+    ids_from_dataset_list = dataset_dict.get("saved_views", [])
 
-    # {id: name} in `views` collection
-    sd = {}
-    for saved_view_dict in conn.views.find({"_dataset_id": dataset_id}):
+    # {id: name} in collection_name
+    doc_id_to_name = {}
+    for ref_doc_dict in conn[collection_name].find(
+        {"_dataset_id": dataset_id}
+    ):
         try:
-            sd[saved_view_dict["_id"]] = saved_view_dict["name"]
+            doc_id_to_name[ref_doc_dict["_id"]] = ref_doc_dict["name"]
         except:
             pass
 
     # Make sure docs in `views` collection match IDs in `dataset_dict`
-    saved_view_ids = set(saved_views)
-    saved_view_doc_ids = set(sd)
+    ids_from_dataset_set = set(ids_from_dataset_list)
```

Comment:
@swheaton there's a few instances of "saved views" nomenclature in this method that need generalizing

Reply:
dang it thought i got em all